src/client/components/pages/parts/privs-edit-modal.js

```diff
@@ -106,6 +115,19 @@ class PrivsEditModal extends React.Component {
 			/>
 		));
 
+		const noteField = this.state.submittable &&
+			<Form.Group>
+				<Form.Label>
+					Note/Reason:
+				</Form.Label>
+				<Form.Control
```

Comment:
We probably need this input to be required.
I don't think there is currently validation on the server-side for an empty note, but we should make it as hard as possible to take admin actions without leaving some reason for it.
So possibly we should throw an error in the server route if the note is empty, as well as enforcing the front-end input?

Reply:
This bit of feedback left.

Reply:
I've made some changes regarding this. I will also make sure to add some more tests for checking this later.